pom.xml

```diff
@@ -210,4 +211,20 @@
       </build>
     </profile>
   </profiles>
+
+  <repositories>
+    <repository>
+      <id>redhat-build</id>
+      <name>Red Hat builds Repository</name>
+      <url>${release.candidate.repository}</url>
+      <layout>default</layout>
+      <releases>
+        <enabled>true</enabled>
+        <updatePolicy>never</updatePolicy>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
```

Comment:
Is there a need for this? I think the booster-parent includes both the Red Hat Maven repos, so this shouldn't be necessary?

Reply:
Not really, it's there just for my (or whoever else would develop with product bits) convenience so when I am testing the product bits, I can just set the `release.candidate.repository` property to the candidates repo. Could easily be deleted though.

Reply:
Well that's a very good point, I had to do quite some gymnastics to run test against staging repositories! This would be a huge simplification.

Could we maybe have a generic solution in the booster-parent? A profile that would be activated by a presence of a property, say, `extra.maven.repository`, and would configure and additional `<repository>` and `<pluginRepository>`?

Reply:
hmm, that's a good idea. @cescoffier what do you think?

Reply:
I agree. We can move that into the booster parent. Fancy a PR?

Reply:
https://github.com/openshiftio/booster-parent/pull/24

Reply:
Thanks @spisiakm!